api/mfx_dispatch/linux/device_ids.h

```diff
@@ -0,0 +1,475 @@
+// Copyright (c) 2021 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <stdio.h>
+#include <sys/stat.h>
+#include <algorithm>
+#include <string>
+#include <vector>
+
+enum eMFXHWType
+{
+    MFX_HW_UNKNOWN   = 0,
+    MFX_HW_SNB       = 0x300000,
+
+    MFX_HW_IVB       = 0x400000,
+
+    MFX_HW_HSW       = 0x500000,
+    MFX_HW_HSW_ULT   = 0x500001,
+
+    MFX_HW_VLV       = 0x600000,
+
+    MFX_HW_BDW       = 0x700000,
+
+    MFX_HW_CHT       = 0x800000,
+
+    MFX_HW_SKL       = 0x900000,
+
+    MFX_HW_APL       = 0x1000000,
+
+    MFX_HW_KBL       = 0x1100000,
+    MFX_HW_GLK       = MFX_HW_KBL + 1,
+    MFX_HW_CFL       = MFX_HW_KBL + 2,
+
+    MFX_HW_CNL       = 0x1200000,
+
+    MFX_HW_ICL       = 0x1400000,
+    MFX_HW_ICL_LP    = MFX_HW_ICL + 1,
+    MFX_HW_JSL       = 0x1500001,
+    MFX_HW_EHL       = 0x1500002,
+
+    MFX_HW_TGL_LP    = 0x1600000,
+    MFX_HW_RKL       = MFX_HW_TGL_LP + 2,
+    MFX_HW_DG1       = 0x1600003,
+    MFX_HW_ADL_S     = MFX_HW_TGL_LP + 4,
+};
+
+typedef struct {
+    int          device_id;
+    eMFXHWType   platform;
+} mfx_device_item;
+
+// list of legal dev ID for Intel's graphics
```

Comment:
I believe better to remove this comment: "// list of legal dev ID for Intel's graphics". It gives an impression, there are no other legal Intel graphics platfroms

Reply:
Fixed